http-add-on/templates/rbac.yml

```diff
@@ -146,13 +134,7 @@ metadata:
     keda.sh/addon: {{ .Chart.Name }}
     app: {{ .Chart.Name }}
     name: {{ .Chart.Name }}-proxy-role
-    app.kubernetes.io/name: {{ .Chart.Name }}-proxy-role
```

Comment:
As you've mentioned, shouldn't we keep them here? Although your approach is more consistent and better, not sure what to do 🤔

Reply:
Maybe better to align with the operator?

Reply:
This is the last one pending, thoughts @nlamirault

Reply:
I remove `app.kubernetes.io/name` from helper.